Formula/imagemagick.rb

```diff
@@ -1,9 +1,9 @@
 class Imagemagick < Formula
   desc "Tools and libraries to manipulate images in many formats"
   homepage "https://www.imagemagick.org/"
-  url "https://dl.bintray.com/homebrew/mirror/ImageMagick-7.0.9-13.tar.xz"
-  mirror "https://www.imagemagick.org/download/releases/ImageMagick-7.0.9-13.tar.xz"
-  sha256 "c595da7bc601b289a4600cddd962bc666fcbf4bc8fc3c39c80f5c6764002668f"
+  url "https://dl.bintray.com/homebrew/mirror/ImageMagick-7.0.9-14.tar.xz"
+  mirror "https://www.imagemagick.org/download/releases/ImageMagick-7.0.9-14.tar.xz"
+  sha256 "5c45ff1101f0c58a0ca4487959f95c57283638997ac48875c91682ef0432b2e5"
   revision 1
```

Comment:
Could you please delete this line, please, then CI will be happy

Reply:
Done. Thx